src/features/session_trace/aggregate/index.js

```diff
@@ -44,65 +48,131 @@ export class Aggregate extends AggregateBase {
     this.harvestTimeSeconds = getConfigurationValue(agentIdentifier, 'session_trace.harvestTimeSeconds') || 10
     this.maxNodesPerHarvest = getConfigurationValue(agentIdentifier, 'session_trace.maxNodesPerHarvest') || 1000
     this.laststart = 0
+    this.isStandalone = false
+    const operationalGate = new HandlerCache() // acts as a controller-intermediary that can enable or disable this feature's collection dynamically
+    const sessionEntity = this.agentRuntime.session
+
+    /* --- The following section deals with user sessions concept & contains non-trivial control flow. --- */
+    const controlTraceOp = (traceMode) => {
+      switch (traceMode) {
+        case MODE.ERROR:
+        case MODE.FULL:
+        case true:
+          this.startTracing(operationalGate)
+          break
+        case MODE.OFF:
+        case false:
+        default: // this feature becomes "off" (does nothing & nothing is sent)
+          operationalGate.decide(false)
+          break
+      }
+    }
```

Comment:
Is there a reason i'm missing to be overly explicit with the case statements here?

```suggestion
const controlTraceOp = (traceMode) => {
  if (traceMode) this.startTracing(operationalGate)
  else operationalGate.decide(false)
}
```

Reply:
This was originally specific to MODE enum. Might separate true/false apart in the future, or actually have more to-dos for MODE.ERROR, which originally _was_ the case.